samples/official/ai-vision-devkit-get-started/deployment.template.json

```diff
@@ -31,9 +31,10 @@
             "status": "running",
             "restartPolicy": "always",
             "settings": {
-              "image": "mcr.microsoft.com/azureiotedge-hub:1.0",
+              "image": "mcr.microsoft.com/azureiotedge-hub:1.0.4",
```

Comment:
thsi si find is 1.0.4 is getting downloaded now it was not working hence we hard coded .,,,,@Anca can confirm as well

Reply:
We have not seen the access denied on 1.0 node so not sure we need this 1.0.4 hardcoded change, I think below DNS issue should resolve this one also with 1.0 ...